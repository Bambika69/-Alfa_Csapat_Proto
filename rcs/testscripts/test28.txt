# create safehosue with virologist on it and give cloak to virologist and put it on
createfield -t safehouse
createvirologist SA1
createequipment cloak -v V1
startturn V1
toggle E1

# create second safehouse with second virologist and turn him to bear, elements id should be EB4
createfield SA1 -t safehouse
createvirologist SA2
createagent virus bear V2
startturn V2
useagent A1 V2

# create storehouse with third virologist and give him a cloak
createfield SA1 SA2 -t storehouse
createvirologist ST1
createelements n10a10 ns10as10 -s ST1
createequipment gloves -v V3
startturn V3
toggle E2

# create last storehouse field with elements ID should be EB5
createfield SA1 SA2 -t storehouse
createelements n10a10 ns10as10 -s ST2

# move V1 to ST2
startturn V1
move ST2

# move V2 (bear) to ST2, destroy elements, and V1 cloak blocks
setrandom True
startturn V2
move ST2
status EB5