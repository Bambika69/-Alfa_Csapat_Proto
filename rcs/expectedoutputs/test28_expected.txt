Field created:
ID: SA1
Type: Safehouse
Neighbours: null

Virologist created:
ID: V1
Field: SA1
Backpack: B1

Equipment created:
ID: E1
Type: cloak
Destination: V1
Result: Successful

V1's turn started.

Equipment toggled on Virologist:
Equipment: E1
Virologist: V1
Inf: wear


Field created:
ID: SA2
Type: Safehouse
Neighbours: SA1

Virologist created:
ID: V2
Field: SA2
Backpack: B2

Agent created:
ID: A1
Type: virus
GeneticCode: bear
Virologist: V2
Result: Successful

V2's turn started.

Agent used on virologist:
Agent: A1
Virologist: V2
Target: V2
Result: Successful


Field created:
ID: ST1
Type: Storehouse
Neighbours: SA1, SA2

Virologist created:
ID: V3
Field: ST1
Backpack: B3

Elements created:
ID: EB4
Nucleotide: 10
AminoAcid: 10
NucleotideSize: 10
AminoAcidSize: 10
Destination: ST1

Equipment created:
ID: E2
Type: gloves
Destination: V3
Result: Successful

V3's turn started.

Equipment toggled on Virologist:
Equipment: E2
Virologist: V3
Inf: unwear


Field created:
ID: ST2
Type: Storehouse
Neighbours: SA1, SA2

Elements created:
ID: EB5
Nucleotide: 10
AminoAcid: 10
NucleotideSize: 10
AminoAcidSize: 10
Destination: ST2


V1's turn started.

Virologist moved:
Virologist: V1
Origin: SA1
Destination: ST2


Random behaviour changed:
yesOrNoType: True
choiceType: Random


